cves/kernel/CVE-2020-9391.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: An issue that was seen in the 5.4 version of the Linux kernel and 5.5 
+  of the AArch64 (ARM64) architechture, which it part of the ARM architecture family.
+  The issue causes the top byte of the address passed to the brk() system call to be ignored.
+  This can potentially cause the memory break to move downward instead of upwards. The result
+  is a heap curruption.
```

Comment:
A brief explanation of what heap corruption is could help those who don't know. Aside from that, looks good. 